dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -857,6 +945,7 @@ def assert_caching_enabled(cache_control_header, max_age, proxy_max_age)
 
     assert_select '#notStarted'
     assert_select '#codeApp', 0
+    refute_includes response.body, fake_last_attempt
```

Comment:
Before, the assertions in this `does not load applab` test were _exactly_ the same as the ones in the `loads applab` tests, above.  Seems like it wasn't testing what it meant to test at all?  We think this last_attempt check helps with that - let us know if you disagree.